### PR TITLE
 Pushjet support (attempt #2) :)

### PR DESCRIPTION
**Pull request by paradix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +88,7 @@ should be known that Notify.py depends on the following packages:
 | pyOpenSSL **[P]**            | 16.2.0  | https://pypi.python.org/pypi/pyOpenSSL/16.2.0                                          |
 | gntp                         | 1.0.2   | https://pypi.python.org/pypi/gntp/1.0.2                                              |
 | urllib3 **[P]**              | 1.12    | https://pypi.python.org/pypi/urllib3/1.12                                            |
+| pushjet                      | 1..0.0    | https://pypi.python.org/pypi/pushjet/1.0.0                                           |
```
Comment (caronc) [inline README.md]: minor: double period in version

File: Notify/pushjet/pushjet.py [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+                "This should probably be reported to: "
+                "https://github.com/Pushjet/Pushjet-Server-Api/issues"
+            )
+        try:
+            response = r.json()
+        except ValueError:
+            response = {}
+        return status, response
+
+default_api = Api(DEFAULT_API_URL)
```
Comment (caronc) [inline Notify/pushjet/pushjet.py]: Just curious... this code gets initialized every time your code is sourced (even for those not using this plugin).  Is this necessary?

I was also curious, i noticed the content in _Notify/pushjet_ has no license associated with it.  Is this your code? Or is there a source (to credit)?

File: Notify/pnotify/NotifyPushjet.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                if self.port:
+                    server += ":" + str(self.port)
+
+                api = pushjet.Api(server)
+                service = api.Service(secret_key=self.user)
+            else:
+                service = pushjet.Service(secret_key=self.host)
+
+            service.send(body, title)
+        except errors.PushjetError as e:
```
Comment (caronc) [inline Notify/pnotify/NotifyPushjet.py]: This code works great! Just this line can fail to catch ValueError which is thrown in the **pushjet.pushjet.Service** **__init__()** function.  I was testing different variations (bad keys and just the fact i don't have a server running and that seems to be the only unhandled exception.  Great job!

Maybe just change this line from:
`except errors.PushjetError as e:`

to
`except (errors.PushjetError, ValueError) as e:`

That should bullet-proof that part

**Comment by paradix** [comment]
The pushjet code is provided by Pushjet itself as a module, so license has to be added for credit.
But the other thing is, it also depends on the requests module (https://pypi.python.org/pypi/requests).
Do you think it's still ok when it's added to the dependencies?

**Pull request merged**